openmetadata-ui/src/main/resources/ui/src/components/MlModelVersion/MlModelVersion.test.tsx

```diff
@@ -0,0 +1,250 @@
+  () =>
+    jest
+      .fn()
+      .mockImplementation(() => (
+        <div data-testid="DataAssetsVersionHeader">DataAssetsVersionHeader</div>
+      ))
+);
+
+jest.mock('components/TabsLabel/TabsLabel.component', () =>
+  jest
+    .fn()
+    .mockImplementation(({ name }) => (
+      <div data-testid={`TabsLabel-${name}`}>{name}</div>
+    ))
+);
+
+jest.mock('components/Tag/TagsContainerV2/TagsContainerV2', () =>
+  jest
+    .fn()
+    .mockImplementation(() => (
+      <div data-testid="TagsContainerV2">TagsContainerV2</div>
+    ))
+);
+
+jest.mock('components/common/CustomPropertyTable/CustomPropertyTable', () => ({
+  CustomPropertyTable: jest
+    .fn()
+    .mockImplementation(() => (
+      <div data-testid="CustomPropertyTable">CustomPropertyTable</div>
+    )),
+}));
+
+jest.mock('components/common/description/DescriptionV1', () =>
+  jest
+    .fn()
+    .mockImplementation(() => (
+      <div data-testid="DescriptionV1">DescriptionV1</div>
+    ))
+);
+
+jest.mock('components/common/error-with-placeholder/ErrorPlaceHolder', () =>
+  jest
+    .fn()
+    .mockImplementation(() => (
+      <div data-testid="ErrorPlaceHolder">ErrorPlaceHolder</div>
+    ))
+);
+
+jest.mock('components/EntityVersionTimeLine/EntityVersionTimeLine', () =>
+  jest
+    .fn()
+    .mockImplementation(() => (
+      <div data-testid="EntityVersionTimeLine">EntityVersionTimeLine</div>
+    ))
+);
+
+jest.mock('components/VersionTable/VersionTable.component', () =>
+  jest
+    .fn()
+    .mockImplementation(() => (
+      <div data-testid="VersionTable">VersionTable</div>
+    ))
+);
+
+jest.mock('components/Loader/Loader', () =>
+  jest.fn().mockImplementation(() => <div data-testid="Loader">Loader</div>)
+);
+
+jest.mock('react-router-dom', () => ({
+  useHistory: jest.fn().mockImplementation(() => ({
+    push: mockPush,
+  })),
+  useParams: jest.fn().mockReturnValue({
+    tab: 'container',
+  }),
+}));
+
+describe('TableVersion tests', () => {
+  it('Component should render properly when not loading', async () => {
+    await act(async () => {
+      render(<MlModelVersion {...mlModelVersionMockProps} />);
+    });
+
+    const dataAssetsVersionHeader = screen.getByTestId(
+      'DataAssetsVersionHeader'
+    );
+    const description = screen.getByTestId('DescriptionV1');
+    const featureTabLabel = screen.getByTestId(
+      'TabsLabel-label.feature-plural'
+    );
+    const customPropertyTabLabel = screen.getByTestId(
+      'TabsLabel-label.custom-property-plural'
+    );
+    const entityVersionTimeLine = screen.getByTestId('EntityVersionTimeLine');
+
+    expect(dataAssetsVersionHeader).toBeInTheDocument();
+    expect(description).toBeInTheDocument();
+    expect(featureTabLabel).toBeInTheDocument();
+    expect(customPropertyTabLabel).toBeInTheDocument();
+    expect(entityVersionTimeLine).toBeInTheDocument();
+  });
+
+  it('Only loader should be visible when the isVersionLoading is true', async () => {
```

Comment:
Can we make sure that we are following standard test case titles?

should render / display <something> if <something happens> / <some conditions>